src/util/constants.ts

```diff
@@ -20,3 +20,6 @@ export const TAB = "\u200B \u200B \u200B" as const;
 
 export const EMOJI_NEWBIE = "<:newbie:962332319623049226>" as const;
 export const ASSISTCHANNELS = ["986520997006032896", "998942774994927646"];
+
+export const URL_REGEX =
+	/(?<protocol>https?:\/\/(?:www\.|(?!www))[\dA-Za-z][\dA-Za-z-]+[\dA-Za-z]\.\S{2,}|www\.[\dA-Za-z][\dA-Za-z-]+[\dA-Za-z]\.\S{2,}|https?:\/\/(?:www\.|(?!www))[\dA-Za-z]+\.\S{2,}|www\.[\dA-Za-z]+\.\S{2,})/g;
```

Comment:
This capture group is not the protocol!

```suggestion
	/(?<url>https?:\/\/(?:www\.|(?!www))[\dA-Za-z][\dA-Za-z-]+[\dA-Za-z]\.\S{2,}|www\.[\dA-Za-z][\dA-Za-z-]+[\dA-Za-z]\.\S{2,}|https?:\/\/(?:www\.|(?!www))[\dA-Za-z]+\.\S{2,}|www\.[\dA-Za-z]+\.\S{2,})/g;
```